irohad/main/application.cpp

```diff
@@ -41,7 +41,9 @@ Irohad::Irohad(const std::string &block_store_dir,
       pg_conn_(pg_conn),
       torii_port_(torii_port),
       internal_port_(internal_port),
-      keypair(keypair) {
+      keypair(keypair),
+      test_storage_(TestStorageImpl::create(
```

Comment:
Why using test storage ?

Reply:
Storage cannot drop database while `TestStorage` can. Probably it should be renamed to `DroppableStorage`